setup.py

```diff
@@ -20,11 +21,10 @@
     install_requires=[
         'six>=1.10.0,<2.0.0',
         'stevedore>=0.14.1,<2.0.0',
-        'pymongo>=2.7.2,<4.0.0'
+        'pymongo>=2.7.2,<4.0.0',
+        'Django>=1.11,<2.0;python_version<"3"',
+        'Django>=1.11;python_version>"3"'
```

Comment:
Django is an optional dependency for this package, so it has to stay in `extras_require` instead of `install_requires`.  You can just leave it as `Django>=1.11` if getting the environment markers to work correctly in `extras_require` proves difficult.